tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -509,3 +508,12 @@ Other free text and markdown formatting can be used elsewhere in the document if
 
 ### [shares are not listed with full paths](https://github.com/owncloud/ocis/issues/2462)
 -   [webUISharingPublicBasic/publicLinkCreate.feature:88](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature#L88)
+
+### [web config update is not properly reflected after the ocis start](https://github.com/owncloud/ocis/issues/2944)
+-   [webUIFiles/breadcrumb.feature:50](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFiles/breadcrumb.feature#L50)
+
+### Shares folder cannot be deleted and is always visible
```

Comment:
I don't think the `Shares` folder is supposed to be delete-able so why bother running these tests? Might make more sense to exclude them from being run on oCIS instead

Reply:
These were already added to expected-failures in the oCIS repo in commit:
https://github.com/owncloud/ocis/commit/f542e58f773d946ab9a020e75c61b0a7efd09a09
and
https://github.com/owncloud/ocis/commit/1129d3c53d57a110c6a633839df9460b26f08873

This is the same items being added here. IMO for consistency we should make the same additions here. And also we should raise an issue about this, to look into why these tests really started failing. They started failing with https://github.com/owncloud/ocis/pull/2739 "Spaces registry".

If it really is a genuine feature change, then we can make a separate PR to skip the relevant test scenarios on oCIS (and possibly make a scenario to demonstrate the expected behavior on oCIS and skip it on oC10). And that will get rid of these entries in expected-failures here in web. Then we bump the web commit-id in oCIS, and the entries will go from there also.

If I had realized this earlier, we could have sorted out the tests already.

Reply:
Thanks for the discussion.
The `Shares` folder is not supposed to be deleted and trying to do so should fail. As far as I understood, expected failures are for "things that fail but will be fixed in the future". If we already know that the test will never succeed (which is the case at least for ocis, for oc10 I'm not sure), skipping them entirely would be best.

@phil-davis Do you guys have some documentation available for "what belongs in expected failures and what not"? To point people to a sort of guide of how to deal with that? Shared understanding is important here IMO.

Reply:
IMO we don't need tests that try to delete the Shares folder. I agree that we should keep the files expected failures files consistent. But for what it's worth, I'd like to get the tests deleted (can happen in a followup PR). Sufficient to test this on the API in my opinion. As long as the `Shares` folder has no delete permissions the UI should behave correctly and in my opinion it would be desirable to have only unit tests for the web ui regarding permission checks.